feed.go

```diff
@@ -202,6 +196,7 @@ func (feed *ArtifactsFeed) GetStableRelease(series string) (*ArtifactVersion, er
 	}
 
 	for _, version := range feed.Versions {
+		// Will current be accurate for the new versioning scheme?
```

Comment:
temporary comment to highlight my confusion

Reply:
This should still be correct.